htdocs/admin/notification.php

```diff
@@ -198,6 +202,20 @@
 }
 print '</td>';
 print '</tr>';
+
+$arrayofnotifto = array(
+'',
+$langs->trans('notiftouser'),
+$langs->trans('notiftofixedemail'),
+$langs->trans('notiftouserandtofixedemail')
+);
+
+print '<tr class="oddeven"><td>';
+print $langs->trans("NotificationDisableConfirmMessage").'</td>';
+print '<td>';
+print $form->selectarray("notif_disable", $arrayofnotifto, $conf->global->NOTIFICATION_EMAIL_DISABLE_CONFIRM_MESSAGE, 0, 0, 0, '', 1, 0, 0, '', 'maxwidth400', 1);
```

Comment:
My understanding is that you want an option to disable globally any notification to users/contacts
And also an option to disable globally any notification to fixed emails ?
If right, 2 differents var would be better.
Each var can be set to yes or no.

Reply:
@eldy Ok i have change method